packages/documentation/src/pages/getting-started/common-tasks/new-end-to-end-test.mdx

```diff
@@ -0,0 +1,153 @@
+
+## end-to-end testing overview
+
+- `vets-website` uses [Nightwatch](https://nightwatchjs.org) to run the tests and provide the browser client
+- end-to-end tests are **collocated in application folder** with the application they test
+- Two node apps run with the end-to-end tests:
+  - `mockapi.js` - hosts mock responses (see [Mocking API responses](#mocking-api-responses))
+  - `test-server.js` - builds and hosts `vets-website` with support for client-side routes
+
+## end-to-end tests conventions
+
+**00-main-test-file.e2e.spec.js**
+```
+const E2eHelpers = require('platform/testing/e2e/helpers')
+const Timeouts = require('platform/testing/e2e/timeouts.js');
+const MyApplicationHelpers = require('./my-application-helpers.js');
+const testData = require('./schema/maximal-test.json');
+const FormsTestHelpers = require('platform/testing/e2e/form-helpers');
+
+// export the test using E2eHelpers.createE2eTest
+module.exports = E2eHelpers.createE2eTest(client => {
+  // mock api responses
+  MyApplicationHelpers.initApplicationSubmitMock();
+  MyApplicationHelpers.initEnrollmentStatusMock();
+
+  // use a comment to indicate what page is being tested
+  // Introduction page
+  client
+    .openUrl(`${E2eHelpers.baseUrl}/my-application`)
+    // use Timeouts constants
+    .waitForElementVisible('body', Timeouts.normal)
+    .assert.title('My Application Title | Veterans Affairs')
+    .click('.schemaform-start-button');
+
+  // disable scrolling
+  FormsTestHelpers.overrideFormsScrolling(client);
+
+  // assert navigation is successful
+  E2eHelpers.expectNavigateAwayFrom(client, '/introduction');
+
+  // Personal Information page.
+  client.expect.element('input[name="root_veteranFullName_first"]').to.be
+    .visible;
+  // use functions from the helper file to perform all actions on the page
+  MyApplicationHelpers.completePersonalInformation(client, testData.data);
+  client.axeCheck('.main').click('.form-panel .usa-button-primary');
+  E2eHelpers.expectNavigateAwayFrom(
+    client,
+    '/my-application/personal-information',
+  );
+  ```
+
+**my-application-helpers.js**
+```
+const mock = require('platform/testing/e2e/mock-helpers');
+const Timeouts = require('platform/testing/e2e/timeouts.js');
+const Auth = require('platform/testing/e2e/auth.js');
+const moment = require('moment');
+
+function completePersonalInformation(client, data) {
+  client
+    .waitForElementVisible(
+      'input[name="root_veteranFullName_first"]',
+      Timeouts.normal,
+    )
+    .fill(
+      'input[name="root_veteranFullName_first"]',
+      data.veteranFullName.first,
+    )
+    .fill('input[name="root_veteranFullName_last"]', data.veteranFullName.last)
+}
+```
+_These are recommendations not requirements._
+
+- separate navigation from field input
+  - use a **main test file** for navigation, assertions, and calls helpers
+  - use a **helper file** for filling out forms
+- create separate, numbered **main test files** to organize tests by their focus:
```

Comment:
I don't we're this intentional, I think the current naming is held over from previous ways of organizing tests. We might want to leave out anything but very basic guidelines until we have a coherent suggestion for naming.